azurelinuxagent/ga/exthandlers.py

```diff
@@ -985,9 +1012,13 @@ def on_rmtree_error(_, __, exc_info):
         # Also remove the cgroups for the extension
         CGroupConfigurator.get_instance().remove_extension_cgroups(self.get_full_name())
 
-    def update(self, version=None):
+    def update(self, version=None, disable_failed=False):
         if version is None:
             version = self.ext_handler.properties.version
+        env = {'VERSION': version}
+
+        if disable_failed:
+            env.update({DISABLE_FAILED: "1"})
```

Comment:
Can we use `True` instead? A number might indicate we're counting the number of failures instead of setting a flag. Same comment applies for other assignments.

Reply:
We specifically wanna use a number to avoid the confusion of case-sensitive value vs case-insensitive because you can only set a string value for environment variables.